### PR TITLE
[4.0] Wrap the button group in quick icons

### PR DESCRIPTION
**Pull request by chmst**
Pull Request for Issue #25966 .

### Summary of Changes
Wrap the button group in a < li

### CODE REVIEW

File: administrator/templates/atum/scss/blocks/_quickicons.scss [hunk 1]
```diff
@@ -15,6 +15,10 @@
     margin: 0.5rem;
     padding: 0;
 	
+    > ul {
```
Comment (C-Lodder) [inline administrator/templates/atum/scss/blocks/_quickicons.scss:14]: `.list-unstyled` does the same...

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 4b6605a37b40cb599f3adef27fefd0747a1dd5dd<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26167">issues.joomla.org/tracker/joomla-cms/26167</a>.</sub>

**Comment by brianteeman** [comment]
this solves the accessibility issue and the invalid markup - thanks

**Comment by ChristineWk** [comment]
result of patch/check of html, if it's OK?

screen shot 2019-09-07 at 12 20 33<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26167">issues.joomla.org/tracker/joomla-cms/26167</a>.</sub>

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on 0c661f25adfb6a4e22f847b9ab17cc130a6e6a76<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26167">issues.joomla.org/tracker/joomla-cms/26167</a>.</sub>

**Comment by ghost** [comment]
Status "Ready To Commit". 1 Build is failing.

**Comment by zero-24** [comment]
Should't the comment by @C-Lodder checked before we move this to RTC?

File: administrator/templates/atum/scss/blocks/_quickicons.scss (hunk 1 above)
Comment (chmst) [reply administrator/templates/atum/scss/blocks/_quickicons.scss:14]: done

**Comment by chmst** [comment]
Thank you @C-Lodder for the hint.

**Comment by C-Lodder** [comment]
The CSS is still there. It can be removed

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**